tests/remote/remote_controller.pm

```diff
@@ -19,6 +19,8 @@ sub run() {
     my $target_ip;
 
     select_console 'root-console';
+    type_string "chown $username /dev/$serialdev\n";
```

Comment:
good point by @sysrich . My general recommendation is call `assert_script_run` by default in text console test modules. We have some cases where we "don't really care" about the command success but then wonder later on about weird test symptoms and confusing error messages that have always been there because no one checks the result of the commands. Calling `chown` should always succeed regardless if it has been called in before.